apps/03TicTacToe.js

```diff
@@ -22,31 +22,79 @@ function printBoard() {
 }
 
 function horizontalWin() {
-    // Your code here
+  if((board[0][0] === playerTurn &&
+    board [0][1]=== playerTurn &&
+    board[0][2] === playerTurn)||
+    (board[1][0] === playerTurn &&
+    board [1][1] === playerTurn &&
+    board[1][2] === playerTurn)||
+    (board[2][0] === playerTurn &&
+    board [2][1] === playerTurn &&
+    board[2][2] === playerTurn)) {
+    return true;
```

Comment:
I like whitespace. Nice. Remember when you `if (condition) { return true } else { return false }`, you can clean it up and just say `return condition`